.github/zizmor.yml

```diff
@@ -0,0 +1,5 @@
+rules:
+  dangerous-triggers:
+    ignore:
+      # this workflow is only triggered after maintainer approval
+      - upload_pr_documentation.yml:3:1
```

Comment:
It's not beautiful but I don't know if there is anything else that we can do here.

Reply:
The [zizmor documentatin](https://woodruffw.github.io/zizmor/audits/#dangerous-triggers) recommends switching to the `workflow_call` trigger which has the security benefit of not passing environment variables from one workflow to the other - they're more isolated.

FWIW it seems to be the better scheme for this workflow anyway but I'm not sure if there are unforeseen consequences.